### PR TITLE
add instructions to autoscale aws

### PR DESCRIPTION
**Pull request by iffyuva**

### CODE REVIEW

File: selfhosting/auto-scaling-aws.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# AWS EKS auto scaling instructions
+
+Unlinke GCP, AWS EKS doesn't enable autoscaling by default. You need to install
+[autoscaler][k8s-autoscaler] specific to AWS EKS cluster. Thanks to helm, we
+can install cluster-autoscaler chart. First prepare iam roles and node groups
+as suggested in readme of [helm chart][helm-cluster-autoscaler]. Steps to
+follow:
```
Comment (roguesherlock) [inline selfhosting/auto-scaling-aws.md]: > First prepare iam roles and node groups
 as suggested in readme of [helm chart][helm-cluster-autoscaler]

Does this line imply that user has to manually read the readme and do the steps mentioned there ? I think not.

File: selfhosting/auto-scaling-aws.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+
```
Comment (roguesherlock) [inline selfhosting/auto-scaling-aws.md:8]: Should we also add a link to our set up eks instructions ?

**Review by roguesherlock — COMMENTED** [review]
looks good

File: selfhosting/auto-scaling-aws.md (hunk 1 above)
Comment (iffyuva) [reply selfhosting/auto-scaling-aws.md]: Oh yeah, no. Need to change this.

File: selfhosting/auto-scaling-aws.md (hunk 2 above)
Comment (iffyuva) [reply selfhosting/auto-scaling-aws.md:8]: Makes sense. Wondering how that would be in gitbooks?

Comment (roguesherlock) [reply selfhosting/auto-scaling-aws.md:8]: Will have to manually update the link in git book. That’s fine

**Review by roguesherlock — APPROVED** [review]

File: selfhosting/auto-scaling-aws.md (hunk 2 above)
Comment (sreecodeslayer) [reply selfhosting/auto-scaling-aws.md:8]: >Will have to manually update the link in git book. That’s fine

Which is this link, you guys are talking about?

Comment (roguesherlock) [reply selfhosting/auto-scaling-aws.md:8]: https://docs.getdockup.com/how-tos/set-up-dockup-agent-on-aws

**Pull request closed without merging**